e2e/tests/localAuthorityDeletesApplication_test.js

```diff
@@ -0,0 +1,40 @@
+const config = require('../config.js');
+
+let caseId;
+
+Feature('Application draft (empty draft)');
+
+Before(async (I) => {
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+
+    // eslint-disable-next-line require-atomic-updates
+    caseId = await I.grabTextFrom('.heading-h1');
+    console.log(`Application draft ${caseId} has been created`);
+  } else {
+    await I.navigateToCaseDetails(caseId);
+  }
+});
+
+Scenario('local authority tries to submit incomplete case', (I, caseViewPage, submitApplicationEventPage) => {
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  submitApplicationEventPage.giveConsent();
+  I.click('Continue');
+  I.waitForElement('.error-summary-list');
+  I.see('Tell us the status of all documents including those that you haven\'t uploaded');
+  I.see('You need to add details to orders and directions needed');
+  I.see('You need to add details to children');
+  I.see('You need to add details to applicant');
+  I.see('You need to add details to hearing needed');
+  I.see('You need to add details to grounds for the application');
+});
+
+Scenario('local authority deletes application', (I, caseViewPage, deleteApplicationEventPage, eventSummaryPage) => {
```

Comment:
Aren't you introducing order dependency in those tests now? If for any reason the second test runs first, the first one is going to fail as the created case would be deleted. I generally believe rely on ordering of the tests is bad.

Reply:
I did that on purpose, you can call it design decision. I am aware that generally dependencies between tests are bad thing but here I think I managed to use it to our advantage. The way I see this is that benefits are greater than cost.

Described scenario will not happen in current implementation of framework. Tests are always executed from top to bottom.